src/quo/foundations/resources.cljs

```diff
@@ -5,7 +5,14 @@
    :keycard-chip-light (js/require "../resources/images/ui2/keycard-chip-light.png")
    :keycard-chip-dark  (js/require "../resources/images/ui2/keycard-chip-dark.png")
    :keycard-watermark  (js/require "../resources/images/ui2/keycard-watermark.png")
-   :bored-ape          (js/require "../resources/images/mock2/bored-ape.png")})
+   :bored-ape          (js/require "../resources/images/mock2/bored-ape.png")
+   :no-funds
+   {:light (js/require "../resources/images/ui2/no-funds-light.png")
```

Comment:
should we make a  separate map for themed images ? just to be safe with the usage?